examples/nanocoap_server/Makefile

```diff
@@ -35,7 +35,7 @@ USEMODULE += shell_commands
 # Comment this out to disable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-#CFLAGS += -DDEVELHELP
+DEVELHELP ?= 0
```

Comment:
Only problem here is the same as kaspaar, but it is not shown as he already ACKed

> Nitpick: the comment doesn't make sense if the line says ?=0. Feel free to change the line to #DEVELHELP ?= 1

Reply:
yeah right, missed to adapt that one